From ab7f9662c2ede5b2e2caea517ae2c65380eee998 Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Tue, 27 Dec 2016 15:20:23 +0800
Subject: [PATCH] arm64: dts: rockchip: add PCIe stuff for rk3399-tve1205g

Add PCIe dt-bindings for rk3399-tve1205g and keep them as
disabled status currently.

Change-Id: Ie92c6e330a6588ff88d69306674429d69b971b58
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts b/arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts
index ced31fb5ae66..d5b4b6e74637 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts
@@ -794,6 +794,18 @@
 	status = "disabled";
 };
 
+&pcie_phy {
+	status = "disabled";
+};
+
+&pcie0 {
+	ep-gpios = <&gpio3 7 GPIO_ACTIVE_HIGH>;
+	num-lanes = <4>;
+	pinctrl-names = "default";
+	pinctrl-0 = <&pcie_clkreqn>;
+	status = "disabled";
+};
+
 &pmu_io_domains {
 	status = "okay";
 	pmu1830-supply = <&vcc_1v8>;
-- 
2.35.3

